istioctl/cmd/dashboard.go

```diff
@@ -451,6 +458,7 @@ func dashboard() *cobra.Command {
 		"Address to listen on. Only accepts IP address or localhost as a value. "+
 			"When localhost is supplied, istioctl will try to bind on both 127.0.0.1 and ::1 "+
 			"and will fail if neither of these address are available to bind.")
+	dashboardCmd.PersistentFlags().BoolVar(&skipOpen, "skipOpen", false, "skip open browser or not")
```

Comment:
if your default value is false, how could your dashboard_test.go work?

Reply:
Since it is a bool flag, if you set it, it pass value as `true`:
```
root:[istio]$ go run a.go
Bool Arg: false
root:[istio]$ go run a.go -arg1
Bool Arg: true
root:[istio]$ cat a.go
package main

import (
    "flag"
    "fmt"
)

func main() {
    // Define a bool flag
    boolArgPtr := flag.Bool("arg1", false, "This is a bool argument")

    // Parse command line
    // into the defined flags
    flag.Parse()

    fmt.Println("Bool Arg:", *boolArgPtr)
}
```

Reply:
ok, i'm not familiar with the, but if you tested, it's pass